Relationship rework

hobohime opened the pull request:
Reworked DE_IN_CREASE to allow for some events to now raise both cats affection that wasn't previously accounted for but were worded in a way that implied it. 

Removed some similarly worded relationship events.

Took out "tidy" "dreamy" and "cowardly" and related relationship events.

Changed a few instances of 'blushing' 'blushed' and 'blushes' to cat flavored text. 

Changed "imagines what their kits with (cat) would look like." to "imagines what their family with (cat) would look like.", "wonders what (cat) thinks about kits.", to "wonders what (cat) thinks about mates.", "keeps shyly glancing over at (cat) as the Clan talks about kits." to "keeps shyly glancing over at (cat) as the Clan talks about mates", to take the focus off of reproduction and onto romance. 

Changed "is asking (cat) how babies are made." to "is asking (cat) where kits come from." and adjusted affection gains for it.

Moved "promises (cat) that they'll be mates when they become warriors." to the apprentice_to_others.json and included  "promises to runaway with (cat) if they no longer want to be a medicine cat." as a comparable event for medcat apprentices from warrior apprentices.

hobohime committed:
relationship event changes

Changed a few instances of blushing to cat possible actions, though not all, going through DE_IN_CREASE to make sure every event worded for relationship changes occurs, added a comparable event for medcat apprentices that was available for warrior apprentices, deleted one calm relationship event, changed two instances of "kits" to "mates" to make it accessible for same sex couple events. Made some romantic relationship events more likely to happen. Deleted some events that were similar to others. 'how babies are made' changed to 'where kits come from'.

hobohime committed:
Redid DE_IN_CREASE again

Learnt what relationship.py was doing to increase and decrease affection for both cats, and changed relationship events to react accordingly and appropriately. Made some events that should have been one sided one sided again after making them both sided. Took out "tidy" "cowardly" and "dreamy" and all related events attached to them.

hobohime committed:
Some minor edits

Deleted repeat events and added back in an event I deleted. Cleaned up files to make em pretty

hobohime committed:
Changed how "is asking (cat) where kits come from." effects affection

Decreases comfort for (cat) but increases trust for both sides, and slightly increases comfort for the asker.

Lixxis commented on resources/dicts/relationship_events/special_character.json:
```diff
@@ -29,7 +29,6 @@
     ],
     "calm": [
       "relaxing with (cat).",
-      "is soothing (cat)'s irrational thoughts.",
```
Is there a reason why this is removed?

hobohime replied:
Yes, I forgot include the reason why on my PR. This was removed because the calm personality relationship events are heavily loaded to be emotional labor with this and the 'is helping (cat) calm down." 

I made the decision to remove it and leave the less emotionally charged event in, but I can re-add it if needed.

scribblecrumb replied:
I think it would be better to add, rather than remove.  Perhaps coming up with some less emotional calm events would be preferable.

hobohime replied:
I'll add that back in right now then.

Lixxis commented on resources/dicts/relationship_events/cat_to_other/apprentice_to_other.json:
```diff
@@ -98,7 +99,8 @@
             "promises not to tell (cat)'s mentor that they've been sneaking out of camp.",
-            "playfully teases (cat) for missing a catch."
+            "playfully teases (cat) for missing a catch.",
+            "promises (cat) that they'll be mates when they become warriors."
```
I know there is not currently the possibility for exact romantic interaction between apprentices, so I would suggest removing this string. Because not every close apprentices want to have a mate-relationship with each other.

Lixxis reviewed (changes requested):
Generally there are multiple changes in the IN and DE creasing files, which are not fitting. 

eg: 
removing **"from romantic increase"** for the cat which has the relationship interaction, like: 
* "thinks that (cat) is really charming."
* "might be reading a little too far into (cat)'s kindness."
* "just noticed how beautiful (cat)'s eyes are."

You should redo these changes and look at the original again.

Ryos00 commented on resources/dicts/relationship_events/DE_IN_CREASE/INCREASE_HIGH.json:
```diff
@@ -1,58 +1,43 @@
-            "gives (cat) a wonderfully colored pebble they found, and blushes when thanked.",
-            "gives (cat) a circular Twoleg object they found, and blushes when thanked.",
-            "compliments (cat) on how well groomed their pelt is today.",
+            "gives (cat) a beautiful feather they found, and hides behind their wiskers when thanked.",
```
*whiskers

hobohime committed:
Properly went through love.json

now that I properly understand how relationship increases, I re-went through every line in love.json and attached events that did not have assigned values, undid changes I did when I did not understand, and took out  "promises (cat) that they'll be mates when they become warriors." entirely as well as an event I added.

hobohime committed:
added "is soothing (cat)'s irrational thoughts." back in

Lixxis commented on resources/dicts/relationship_events/DE_IN_CREASE/DECREASE_HIGH.json:
```diff
@@ -20,39 +21,28 @@
             "started a fight with (cat).",
             "has a fight with (cat) about what's right.",
+            "is not backing down in an argument with (cat).",
+            "scorns (apprentice) for not catching enough prey.",
```
I'm not sure if someone changed the code, if not (apprentice) has to be (cat)

hobohime replied:
I'll fix all instances of this right now

Lixxis reviewed (changes requested):
Only one thing I saw, otherwise I think it is alright ! If this is changed or correct @Ryos00 you can merge it.

Ryos00 committed:
Update DECREASE_HIGH.json

hobohime committed:
fixed all instances of (apprentice) and (elder)

They were like that when I started I swear! Now all read (cat)

hobohime committed:
Merge branch 'relationship-rework' of https://github.com/hobohime/clangen into relationship-rework
